storio/src/androidTest/java/com/pushtorefresh/storio/db/integration_test/impl/BaseTest.java

```diff
@@ -102,4 +111,19 @@
 
         return deleteResult;
     }
+
+    void checkNotEmptyAndEquals(@Nullable final List<User> first,
+                                @Nullable final List<User> second) {
+        assertNotNull(first);
+        assertNotNull(second);
+        assertEquals(first.size(), second.size());
+
+        for (User userFromFirst : first) {
+            assertTrue(second.contains(userFromFirst));
```

Comment:
By default `SQLite` will return results in same order what they were stored in the db